src/gui/src/PreferencesDialog.cpp

```diff
@@ -505,9 +505,15 @@ void PreferencesDialog::on_okBtn_clicked()
 	pPref->m_nMidiChannelFilter = midiPortChannelComboBox->currentIndex() - 1;
 
 	//OSC tab
-	pPref->setOscServerEnabled( enableOscCheckbox->isChecked() );
+	if ( enableOscCheckbox->isChecked() != pPref->getOscServerEnabled() ) {
+		pPref->setOscServerEnabled( enableOscCheckbox->isChecked() );
+		H2Core::Hydrogen::get_instance()->toggleOscServer( enableOscCheckbox->isChecked() );
```

Comment:
I just updated and found that these need to be guarded with `#ifdef`s. Trivial fix so I pushed it, I hope you don't mind! :)

Reply:
@cme: It seems that both of us had the same idea there b78a54059f595cbb15dfca26dad4cda55709cb3e :smile:

Reply:
At almost *exactly* the same time! Uncanny1 :D